### ISSUE TITLE
Midpoint Review - Design Feedback - VFS MHV Secure Messaging, MHV Secure Messaging, Secure Messaging

### ISSUE THREAD

**Issue by allison0034** [department-of-veterans-affairs/va.gov-team#51753]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- Why is the emergency alert expanded on the inbox page but not on the reply and compose
- Make sure you follow experimental design process for anything new 
- I would recommend you watch the recording and listen to Matt's feedback. I tried to capture as much of it as I could. The recording is located in the epic under the midpoint review section.

## Feedback
- Practice areas will document their feedback following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback).

**Must:**
- Follow the [navigate a long list styling](https://design.va.gov/patterns/help-users-to/navigate-a-long-list) by loading 5 messages at a time and add “5” into the button text.
- Make sure you are using the [correct icons](https://design.va.gov/foundation/icons) (eg. trash/paperclip)
- For attachment, follow a similar pattern for inline error messaging to [file upload](https://design.va.gov/components/form/file-input)

**Should:**
- Matt Dingee: Rework the attachments ([see Matt’s design](https://sketch.com/s/aee094a9-a0c4-4f08-ba92-cbd4e4c9a43d) using light weight card and icon button)
- Matt Dingee: Rework the tab/segmented menu to a tool bar (see Matt’s design with slimmer icon buttons and gray background)
- Matt Dingee: Since search is limited, search be that folder and advanced search could be a link to a page on its own and change sort to secondary button. If you do want to keep the additional info for search, you may want to rework it slightly to be different visually from additional info (see Matt’s design)
- Matt Dingee: Rework the Inbox to be similar to other mail patterns going to a master list and when you click on a detail you get the message that appears to the right. (see Matt’s design)
- Use inline errors where available instead of modals.

**Consider:**

## Platform directions
- Remove the governance-team label
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Remove the CC-Request label (only if you created the ticket using the New Issue link from the Collaboration Cycle Reviews board)
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by bryan-riley-va** [comment]
@briandeconinck Can you help clarify this MUST feedback?  "Follow the [navigate a long list styling](https://design.va.gov/patterns/help-users-to/navigate-a-long-list) by loading 5 messages at a time and add “5” into the button text."     We are using the long list styling to show 5 results at a time in the message detail view.  Is there another area this refers to?

![MicrosoftTeams-image (7)](https://user-images.githubusercontent.com/105732252/220676405-63bf8e91-6b35-47b3-a76e-e6e299e0365c.png)

**Comment by briandeconinck** [comment]
I believe that feedback was from @allison0034 so I'm going to tag her in!

**Comment by bryan-riley-va** [comment]
This feedback isn't valid anymore based on adjustments that were made to search/filtering: Matt Dingee: Since search is limited, search be that folder and advanced search could be a link to a page on its own and change sort to secondary button. If you do want to keep the additional info for search, you may want to rework it slightly to be different visually from additional info (see Matt’s design)

**Comment by bryan-riley-va** [comment]
Notes on Feedback:

Must:
Follow the [navigate a long list styling](https://design.va.gov/patterns/help-users-to/navigate-a-long-list) by loading 5 messages at a time and add “5” into the button **BR - comment above asking for clarification**
Make sure you are using the [correct icons](https://design.va.gov/foundation/icons) (eg. trash/paperclip) **BR - ticket created in JIRA to validate and address - MHV-**
For attachment, follow a similar pattern for inline error messaging to [file upload](https://design.va.gov/components/form/file-input)   **BR - ticket created in JIRA to validate and address - MHV- **


Should:

Matt Dingee: Rework the attachments ([see Matt’s design](https://sketch.com/s/aee094a9-a0c4-4f08-ba92-cbd4e4c9a43d) using light weight card and icon button) **BR - ticket MHV-41707**
Matt Dingee: Rework the tab/segmented menu to a tool bar (see Matt’s design with slimmer icon buttons and gray background) **BR - comment above asking for clarification**
Matt Dingee: Since search is limited, search be that folder and advanced search could be a link to a page on its own and change sort to secondary button. If you do want to keep the additional info for search, you may want to rework it slightly to be different visually from additional info (see Matt’s design) **BR - redesigned the search/advanced search experience so this one no longer applies**
Matt Dingee: Rework the Inbox to be similar to other mail patterns going to a master list and when you click on a detail you get the message that appears to the right. (see Matt’s design)
 **BR - created future idea ticket to evaluate and change after Phase 0 if needed**
Use inline errors where available instead of modals. **BR - ticket created to address (various tickets)**
